src/pages/Rocket/Rocket.js

```diff
@@ -0,0 +1,53 @@
+import { useDispatch } from 'react-redux';
+import { joinRocket, leaveRocket } from '../../redux/Rockets/Rockets';
+
+const Rocket = (rocket) => {
+  const dispatch = useDispatch();
+
+  const {
+    id,
+    name,
+    image,
+    description,
+    reserved,
+  } = rocket;
```

Comment:
- [ ] Since the rocket data is globally available through the redux store, It would be better not to pass props here. instead, you should use `useSelector` hook. The code will look something like `const rockets = useSelector(state => state.rockets);`
- [ ] The above solution should clear the prop errors displayed in the browser console.

Reply:
Hello @oloomoses I did this but am I am suppose to implement it as an argument or add it as a parameter to the rocket function?

Reply:
This was just an example of what the code should look like. You should replace `state.rocket`  with your data from the store.